package.json

```
@@ -14,23 +14,23 @@
     "@formatjs/intl-pluralrules": "^4.3.2",
     "@metamask/detect-provider": "^1.2.0",
     "@reduxjs/toolkit": "^1.8.0",
-    "@shapeshiftoss/asset-service": "^2.4.5",
-    "@shapeshiftoss/caip": "2.5.0",
-    "@shapeshiftoss/chain-adapters": "^2.15.0",
+    "@shapeshiftoss/asset-service": "^3.0.0",
+    "@shapeshiftoss/caip": "3.0.0",
```

We shouldn't need to lock now that 3.0.0 is stable

```suggestion
    "@shapeshiftoss/caip": "^3.0.0",
```

---

Good catch, fixed up in https://github.com/shapeshift/web/pull/1736/commits/cbedc0daa8f240aecf71a06797854347bb536d85